### DISCUSSION TITLE
Прибавить значение к номеру, который присваиваевается автоматически.

### DISCUSSION THREAD

**Discussion by Leo-ui-528** [totumonline/totum-ru-issues-and-discussions#259] [Как это сделать?] [answered]
Здравствуйте подскажите пожалуйста, как лучше присвоить поле код вида деятельности из таблицы виды деятельности к номеру договора из таблицы циклов, который присваивается автоматически(select(table: $#ntn; field: 'nomer'; order: 'nomer' desc) + 1) и связать его с полем код вида деятельности в документе в цикле? Что бы, когда мы его выбираем при заполнении документа, он присваивался автоматически документу вместе с порядковым номером.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Я так понял вам нужно дописать в номер Договора что-то в зависимости от Кода вида деятельности в цикле.

Я бы это сделал Кодом действия при изменении поля Код вида деятельности:

Код действия:
```
=: set(table: 'cycles'; field: 'number' = str`$number ++ "-" ++ $#nfv`; where: 'id' = $#nci)

number: select(table: 'cycles'; field: 'number'; where: 'id' = $#nci)
```
